io.github.teamclouday.android-mic.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "io.github.teamclouday.android-mic",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "android-mic",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-run/pipewire-0"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/android-mic/cargo"
+    }
+  },
+  "modules": [
+    {
+      "name": "protoc",
+      "buildsystem": "simple",
+      "build-commands": ["install -Dm0755 bin/protoc /app/bin/protoc"],
+      "cleanup": ["*"],
+      "sources": [
+        {
+          "type": "archive",
+          "strip-components": 0,
+          "dest-filename": "protoc.zip",
+          "url": "https://github.com/protocolbuffers/protobuf/releases/download/v29.3/protoc-29.3-linux-x86_64.zip",
+          "sha256": "3e866620c5be27664f3d2fa2d656b5f3e09b5152b42f1bedbf427b333e90021a",
+          "only_arches": ["x86_64"]
+        },
+        {
+          "type": "archive",
+          "strip-components": 0,
+          "dest-filename": "protoc.zip",
+          "url": "https://github.com/protocolbuffers/protobuf/releases/download/v29.3/protoc-29.3-linux-aarch_64.zip",
+          "sha256": "6427349140e01f06e049e707a58709a4f221ae73ab9a0425bc4a00c8d0e1ab32",
+          "only_arches": ["aarch64"]
+        }
+      ]
+    },
+    {
+      "name": "adb",
+      "buildsystem": "simple",
+      "build-commands": ["install -Dm0755 platform-tools/adb /app/bin/adb"],
+      "sources": [
+        {
+          "type": "archive",
+          "strip-components": 0,
+          "dest-filename": "platform-tools-latest-linux.zip",
+          "url": "http://dl.google.com/android/repository/platform-tools-latest-linux.zip",
+          "sha256": "0ead642c943ffe79701fccca8f5f1c69c4ce4f43df2eefee553f6ccb27cbfbe8",
+          "only_arches": ["x86_64"]
```

Comment:
what happens to the aarch64 build of the app without adb? I don't think it's a good idea to build x86_64 and arm64 slightly different.